content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -499,18 +502,25 @@ spec:
 
 {{< feature-state for_k8s_version="v1.9" state="alpha" >}}
 
-To enable this feature in v1.9, the cluster administrator needs to enable the `BlockVolume` feature gate on the apiserver, controller-manager and the kubelet.
+To enable support for raw block volumes, enable the `BlockVolume` feature gate on the
+apiserver, controller-manager and the kubelet.
 
-The following volume plugins support raw block volumes including dynamic provisioning where applicable:
+The following volume plugins support raw block volumes, including dynamic provisioning where
+applicable.
 
 * AWSElasticBlockStore
 * AzureDisk
-* FC (Fibre Channel)
+* FC (Fiber Channel)
```

Comment:
It is actually FibreChannel

Reply:
Wow #TIL! I will raise a follow-on PR for this.